.travis.yml

```diff
@@ -41,6 +41,8 @@ matrix:
     - node_js: "12"
       os: linux
       env: TASK=verify-docs
+      # The default npm for node v12.16.1 (npm v6.13.4) fails with cb() never called
+      before_install: npm i -g npm
```

Comment:
does that mean this line is run in every build?

Reply:
Yes. That’s the travis way of setting npm version. It’s pretty quick.

I also don’t know why other tasks do not fail.